device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/ReportedPropertiesUpdateResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.azure.sdk.iot.device.twin;
+
+import com.microsoft.azure.sdk.iot.device.IotHubStatusCode;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * The response fields for a reported properties update request.
+ */
+@AllArgsConstructor
+public class ReportedPropertiesUpdateResponse
+{
+    /**
+     * The status code returned by the service for this request. May indicate failure to send the reported properties.
+     */
+    @Getter
+    private final IotHubStatusCode statusCode;
+
+    /**
+     * The new version of the reported properties after a successful reported properties update.
+     */
+    @Getter
+    private final int version;
```

Comment:
We had to add a special note in the .NET sdk that edge module property update requests would not have the updated value in here, since edge performed the actual update offline. Would that be relevant for the java sdk as well?

Reply:
Sounds like a good idea, yeah. Where is this note in the .NET SDK?

Reply:
https://github.com/Azure/azure-iot-sdk-csharp/blob/preview/iothub/device/src/ClientPropertiesUpdateResponse.cs#L28